protokolle/2020-06-06_htw-pv3_postgresql_II.md

```diff
@@ -0,0 +1,139 @@
+# 2020-05-06 14:00 - 15:30 Wetterdatenverarbeitung mit PostgreSQL II -----#5
+
+#Auswertung Abgaben Aufgaben #1 und #2
+
+**Bewertungen**
+	* Kriterien
```

Comment:
remove the tabstops before first order list elements

Remove them in the whole document, otherwise it looks like this:

![grafik](https://user-images.githubusercontent.com/46815982/81380387-1fa81b80-910b-11ea-85e8-536f073db7e2.png)

Without tabstop it looks like this:

![grafik](https://user-images.githubusercontent.com/46815982/81380457-3f3f4400-910b-11ea-837d-e53e59c8f906.png)

And if you want multiple orders you can have tabstops before list elemts but not the first elemet! See the [MD sheet](https://github.com/htw-pv3/literature/blob/master/crib-cheat_markdown.md)